test/unit/utils.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+require('colors')
```

Comment:
nope. you don't need it. once `colors` is required, it modifies the string prototype globally. since you require `utils` here, it will require `colors` for you, and you don't need to require it here. This is helpful so we don't have `require('colors')` all over the code...

I just deleted this line and ran the unit tests myself. You don't need it. :smile: